lib/smart_answer_test_helper.rb

```diff
@@ -11,10 +11,24 @@ def self.responses_and_expected_results
     Dir[data_path.join('*-responses-and-expected-results.yml')]
   end
 
+  def self.configurations
+    yaml = File.read(data_path.join('configurations.yml'))
+    YAML.load(yaml)
+  end
+
+  def self.default_configuration
+    configurations.fetch('default')
+  end
+
```

Comment:
Since a bit awkward to make class methods private in Ruby and this is just test code (which will hopefully go away in the not too distant future), I don't think there's much benefit to making these two methods private. In fact I can't see any harm in people calling either of them from the regression test anyway.

However, I will have a look at adding more context to the commit note(s) as you suggest.